### PR TITLE
feat(storage): change docs to use deis-object-storage-secret for configuring the storage

### PR DESCRIPTION
**Pull request by kmala**
change docs to use deis-object-storage-secret for configuring the storage for the builder, slugbuilder , controller and slugrunner.

### CODE REVIEW

File: src/installing-workflow/configuring-object-storage.md [hunk 1]
```diff
@@ -11,26 +11,18 @@ A variety of Deis components rely on an object storage system to do their work.
...
-
-We recommend storing these and all other credentials as Kubernetes secrets. See the below sections for details on configuring each component for details.
-
 # Configuring the Deis Components
 
 Every Deis component that relies on object storage relies on the following two inputs for configuration:
 
-- One or more environment variables that describe what object storage system to use
-- One or more files to provide access credentials for the object storage system.
+- An environment variables that describe what object storage system to use
```
Comment (mboersma) [inline src/installing-workflow/configuring-object-storage.md]: This should say "An environment *variable*" (not plural).

File: src/installing-workflow/configuring-object-storage.md [hunk 2]
```diff
@@ -211,6 +141,8 @@ If the `DATABASE_STORAGE` backend is configured as "s3", the database will recei
 
 ### Google Cloud Storage (Interoperability Mode)
 
+[Google Cloud Storage](https://cloud.google.com/storage/) (GCS) can interoperate with the S3 API using a feature called [interoperability](https://cloud.google.com/storage/docs/interoperability). If you choose to use GCS for object storage for database, you'll have to turn on this interoperability mode. In order to do so, please follow the steps in the [GCS migration documentation](https://cloud.google.com/storage/docs/migrating?hl=en_US#migration-simple).
```
Comment (arschles) [inline src/installing-workflow/configuring-object-storage.md]: does this apply anymore? IIRC all of our codebases support GCS natively...

File: src/installing-workflow/configuring-object-storage.md [hunk 3]
```diff
@@ -11,26 +11,18 @@ A variety of Deis components rely on an object storage system to do their work.
...
-
 # Configuring the Deis Components
 
 Every Deis component that relies on object storage relies on the following two inputs for configuration:
 
-- One or more environment variables that describe what object storage system to use
-- One or more files to provide access credentials for the object storage system.
+- An environment variable that describe what object storage system to use.
+- A configuration file ([objectstorage.toml][objectstorage-toml]) to provide access credentials for the object storage system.
 	- We suggest storing the contents of these files in [Kubernetes secrets][k8s-secret] and mounting them as volumes to each pod
```
Comment (arschles) [inline src/installing-workflow/configuring-object-storage.md]: For consistency with the other bullet points in this section, can you put a `.` character at the end of this bullet point?

File: src/installing-workflow/configuring-object-storage.md (hunk 2 above)
Comment (kmala) [inline src/installing-workflow/configuring-object-storage.md]: Currently i haven't made any changes for the postgres, hence i had moved this from the generic section to database section.

Comment (arschles) [inline src/installing-workflow/configuring-object-storage.md]: Ok, sounds good!

**Comment by arschles** [comment]
Code LGTM, with the exception of the small nitpick at https://github.com/deis/workflow/pull/128#discussion_r57923269

**Pull request merged**